.gitignore

```diff
@@ -1,9 +1,16 @@
-/tests/output/
-/changelogs/.plugin-cache.yaml
+# The following entries have been rationalized and should all have detailed explanations
+
+# Version created and populated by setuptools_scm
+/src/ansible_navigator/_version.py
 
 # tmux integration test logs
 .test_logs
 
+# The following entries came from an untracted source
```

Comment:
This looks like something that would belong in a separate PR.

Reply:
Ok.  Let's see if this is better....

Reply:
@webknjaz LMK if the .gitignore changes are acceptable.

Reply:
Yes, I can live with it, I guess. I'd still prefer a separate gitignore file for folders that actually get checked in.